examples/lighting-app/nrfconnect/README.md

```diff
@@ -85,14 +85,15 @@ with other Thread devices in the network.
 
 ## Requirements
 
-The application requires the nRF Connect SDK v1.5.0 to work correctly.
-
 The example supports building and running on the following devices:
 
 | Board name                                                                                | Board platform build name |
 | ----------------------------------------------------------------------------------------- | ------------------------- |
 | [nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK) | `nrf52840dk_nrf52840`     |
 
+The application requires the most recent supported development version of the nRF Connect SDK to work correctly.
```

Comment:
Now that I think about it, maybe `revision` is better than `version`. Let's discuss at standup.

Reply:
Replaced `versions` with a `specific revision` or `most recent supported revision` depending on context.